### DISCUSSION TITLE
Is it compatible with Katana GF76 - 12UE

### DISCUSSION THREAD

**Discussion by mimosh78** [YoCodingMonster/OpenFreezeCenter#171] [General]
Is it compatible with Katana GF76 - 12UE?
I have not tested the app yet. Just asking to know if it will working on my laptop.

**Comment by YoCodingMonster** [comment]
I have not tested on that specific laptop nor have i recieved anyone posting issues for that model so, cant exactly tell.
Please try the new release of both OpenFreezeCenter and OpenFreezeCenter-Lite and report back if the applications are working
